### ISSUE TITLE
Secure Keyboard input is on when is not on

### ISSUE THREAD

**Issue by kejvi-doko** [withfig/fig#1820]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and tested again (tell us if that fixed it)

### Issue Details

![image](https://user-images.githubusercontent.com/8927481/191965205-f0a8f8d2-2c46-430b-a2ef-77dad3a3508f.png)
![image](https://user-images.githubusercontent.com/8927481/191965265-f1f727fa-c7ae-489d-af43-bf28d7f0ad9c.png)

### Environment

```yaml
fig-details:
  - desktop-version: Version 1.0.61 (B528) [U.S.]
hardware-info:
  - model: MacBook Pro
  - model-id: MacBookPro15,3
  - chip-id: 
  - cores: 8
  - mem: 32 GB
os-info:
  - macOS 12.5.1 (21G83)
environment:
  - shell: /bin/zsh
  - terminal: terminal
  - cwd: /Users/crs
  - exe-path: /usr/local/bin/fig
  - install-method: unknown
  - env-vars:
    - FIG_INTEGRATION_VERSION: 8
    - FIG_TERM: 1
    - FIG_TERM_VERSION: 5.2.0
    - PATH: /Users/crs/.rvm/gems/ruby-2.6.6/bin:/Users/crs/.rvm/gems/ruby-2.6.6@global/bin:/Users/crs/.rvm/rubies/ruby-2.6.6/bin:/Users/crs/.nvm/versions/node/v16.14.2/bin:/usr/local/opt/node@10/bin:/usr/local/opt/postgresql@12/bin:/usr/local/opt/postgresql@12/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/VMware Fusion.app/Contents/Public:/usr/local/go/bin:/usr/local/share/dotnet:~/.dotnet/tools:/Library/Apple/usr/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Users/crs/.fig/bin:/Users/crs/.local/bin:/Users/crs/Library/Android/sdk/emulator:/Users/crs/Library/Android/sdk/tools:/Users/crs/Library/Android/sdk/tools/bin:/Users/crs/Library/Android/sdk/platform-tools:/Users/crs/Library/Python/3.9/bin:/Users/crs/.rvm/bin:/Users/crs/dev/flutter/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - TERM_SESSION_ID: F6B4C4F1-7D62-404E-ACDD-F77D02E2F6F3
    - FIG_PID: 90673
- integrations:
  - SSH: false
  - TMUX: false
  - iTerm: installed!
  - Hyper: application is not present.
  - Visual Studio Code: installed!
  - Docker: true
```

**Comment by mschrage** [comment]
Try following the steps in this guide: https://fig.io/docs/support/secure-keyboard-input

**Comment by kejvi-doko** [comment]
it worked only after I did a restart, thanks.
